src/components/invalidToken/index.tsx

```diff
@@ -0,0 +1,18 @@
+const InvalidToken = () => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        height: '100vh',
+        marginTop: '20vh',
+        justifyContent: 'center',
+        fontSize: 'clamp(20px, 4vw, 42px)',
+        fontFamily: 'monospace',
+      }}
+    >
+      Please provide a valid token!
+    </div>
+  )
+}
+
+export default InvalidToken
```

Comment:
Can we use the next.js error component to define the UI error boundary instead of custom component? With next.js error component, it will be helpful to catch other types of error as well. [See this reference.](https://nextjs.org/docs/app/api-reference/file-conventions/error)